helm/docs/expose-teams-api.md

```diff
@@ -62,6 +62,7 @@ Add a Second Host to the Ingress Controller
 ## Path-Based Routing
 
 Path based routing doesn't require additional DNS entries and TLS certificates.
+In `values.yaml`, remove `apiSettings.dnsName`.
```

Comment:
`apiSettings.dnsName` is not set in the default `values.yaml`

The entire `apiSettings` section is commented out in the overrides file...
```
# apiSettings:
#   Set `dnsName` to expose the API with host-based routing.
#   See https://helm.fiftyone.ai/docs/expose-teams-api.html for more information.
#   dnsName: your-api.hostname.here
```

This instruction is confusing to me as it wouldn't be set unless you'd already set it...

Reply:
Well, if it wasn't set, it would be a no op, right?  Customers shouldn't be editing the chart's `values.yaml`. They should be creating their own values file.

Reply:
most customers use the values file we ship them, which has both that comment and the commented-out-section...